common/flatpak-transaction.c

```diff
@@ -111,10 +111,12 @@ struct _FlatpakTransactionOperation
   GKeyFile                       *resolved_metakey;
   GBytes                         *resolved_old_metadata;
   GKeyFile                       *resolved_old_metakey;
+  char                           *resolved_token;
   guint64                         download_size;
   guint64                         installed_size;
   char                           *eol;
   char                           *eol_rebase;
+  guint32                         need_token;
```

Comment:
In the earlier bits of code, `need_token` was signed. Probably best to be consistent.